src/providers/speak/speak.test.ts

```
@@ -0,0 +1,142 @@
+    start() {
+      mockTus.runUpload(this.options);
+    }
+  },
+}));
+
+jest.mock('react-native-config', () => ({
+  __esModule: true,
+  default: { ECENCY_BACKEND_API: 'https://ecency.test' },
+}));
+
+import { extractPermlink, getUploadTuning, uploadVideoEmbed } from './speak';
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)